src/main/java/de/zalando/tip/zalenium/proxy/DockerSeleniumRemoteProxy.java

```diff
@@ -100,13 +101,28 @@ public int getAmountOfExecutedTests() {
         return amountOfExecutedTests;
     }
 
+    public DockerSeleniumNodePoller getDockerSeleniumNodePollerThread() {
+        return dockerSeleniumNodePollerThread;
+    }
+
     /*
         Class to poll continuously the node status regarding the amount of tests executed. If MAX_UNIQUE_TEST_SESSIONS
         have been executed, then the node is removed from the grid (this should trigger the docker container to stop).
      */
     static class DockerSeleniumNodePoller extends Thread {
 
         private DockerSeleniumRemoteProxy dockerSeleniumRemoteProxy = null;
+        private HttpClient client;
+        private static long sleepTimeBetweenChecks = 500;
+
+        public long getSleepTimeBetweenChecks() {
+            return sleepTimeBetweenChecks;
+        }
+
+        @VisibleForTesting
+        protected void setClient(HttpClient client) {
+            this.client = client;
+        }
 
         public DockerSeleniumNodePoller(DockerSeleniumRemoteProxy dockerSeleniumRemoteProxy) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this constructor to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1213)